### PR TITLE
file tests added for IsSummary Check

### PR DESCRIPTION
**Pull request by Tejesh-SH**

### CODE REVIEW

File: ETWAnalyzer_uTest/EventDump/DumpFileTests.cs [hunk 1]
```diff
@@ -36,6 +38,19 @@ public class DumpFileTests
             FileRenameCount = 150,
         };
 
+        public List<string> Messages { get; set; } = new List<string>();
+
+        private ITestOutputHelper myWriter;
+        public DumpFileTests(ITestOutputHelper myWriter)
```
Comment (AloisKraus) [inline ETWAnalyzer_uTest/EventDump/DumpFileTests.cs:44]: Add a newline between the field declarations and the ctor definition.

File: ETWAnalyzer_uTest/EventDump/DumpFileTests.cs [hunk 2]
```diff
@@ -504,6 +519,145 @@ public void MinMaxSizeFilter()
...
+            Console.SetOut(writer);
+            fileDumper.PrintData(data);
+            testOutput.Add(writer.ToString());
+            string[] lines = writer.ToString().Split(myNewLineSplitChars, StringSplitOptions.RemoveEmptyEntries);
+            Assert.Equal(8, lines.Length);
+            Assert.Contains("Process Count: 2", lines[7]);
+            Assert.Contains("TotalTime: 0.00000 s", lines[7]);
+            Assert.Contains("File/s Total with 0 accessed file/s", lines[7]);
+        }
+
```
Comment (AloisKraus) [inline ETWAnalyzer_uTest/EventDump/DumpFileTests.cs:660]: We should have also a test where ShowTotal = null and only one row is printed where totals are then also omitted. Like a test case OmitSummaryIfOnlyOneLineIsPrinted()

**Review by AloisKraus — COMMENTED** [review]
One more test would be nice and some line spacings to make the Arrange/Act/Assert pattern of each test more visually apparent.

File: ETWAnalyzer_uTest/EventDump/DumpFileTests.cs [hunk 3]
```diff
@@ -642,14 +653,44 @@ public void PrintIsSummaryTotalsNullFileMode()
...
             };
+
             StringWriter writer = new();
             Console.SetOut(writer);
+
             fileDumper.PrintData(data);
+
             testOutput.Add(writer.ToString());
             string[] lines = writer.ToString().Split(myNewLineSplitChars, StringSplitOptions.RemoveEmptyEntries);
             Assert.Equal(8, lines.Length);
```
Comment (AloisKraus) [inline ETWAnalyzer_uTest/EventDump/DumpFileTests.cs]: There should be a negative assert that no substring with the string "File/s Total" is present.
